app/views/shared/_navbar.html.erb

```diff
@@ -61,17 +53,16 @@
       <% end %>
 
       <!-- Button (call-to-action) -->
-      <%= link_to t(".top_call_to_action", default: "Créer un groupe"), "#", class: "navbar-wagon-item navbar-wagon-button btn" %>
+      <%= link_to t(".top_call_to_action", default: "Créer un groupe"), dashboard_path, class: "navbar-wagon-item navbar-wagon-button btn" %>
```

Comment:
Dans l'idée, je suis d'accord mais j'aimerais trouver un truc de plus sexy qui soit un vrai call to action: "verbe" ou "verbe + complément"...il ne me vient pas pour l'instant
Let's move on....mais il faudrait le mettre en français..."Bougeons nous ", un peu trop agressif
Pas inspirée ce matin...une idée?